feat: Introduce Helm chart for Azure Cosmos DB external scaler (#203)
kedacore/charts: CONTRIBUTING.md

# Contributing to KEDA Helm Charts

Thanks for helping making KEDA better!

## Shipping a new version

You can easily release a new Helm chart version:

1. Update the version of the Helm chart in `Chart.yaml`.

> 1. Package the Helm chart

Reviewer: ```suggestion
2. Package the Helm chart
```

    - For KEDA:
        ```shell
        $ helm package keda --destination docs
        Successfully packaged chart and saved it to: docs/keda-2.4.0.tgz
        ```
    - For HTTP Addon:
        ```shell
        $ helm package http-add-on --destination docs
        Successfully packaged chart and saved it to: docs/keda-add-ons-http-0.2.0.tgz
        ```
    - For KEDA external scaler for Azure Cosmos DB:
        ```shell

>         $ helm package cosmosdb-scaler --destination docs
>         Successfully packaged chart and saved it to: docs/keda-external-scaler-azure-cosmos-db-0.1.0.tgz

Reviewer: This does not seem correct as the output name should use the name of the folder.

That said, we should rename the folder as well:
```suggestion
        $ helm package azure-cosmos-db-add-on --destination docs
        Successfully packaged chart and saved it to: docs/azure-cosmos-db-add-on-0.1.0.tgz
```

        ```

> 1. Re-index the Helm repo to add our new version:

Reviewer: ```suggestion
3. Re-index the Helm repo to add our new version:
```

    ```shell
    $ helm repo index docs --url https://kedacore.github.io/charts
    ```

> 1. Commit changes:

Reviewer: ```suggestion
4. Commit changes:
```

1. Re-index the Helm repo to add our new version:
    ```shell
    $ helm package keda
    Successfully packaged chart and saved it to: C:\Code\GitHub\charts\keda-0.1.0.tgz
    $ helm repo index docs --url https://kedacore.github.io/charts
    ```
   - For HTTP Addon:
1. Commit changes:
    ```shell
    $ helm package http-add-on
    Successfully packaged chart and saved it to: /home/ecomaz/src/keda/charts/keda-add-ons-http-0.2.0.tgz
    git add .
    git commit -sm "Packaged new Helm chart version"
    git push origin chart-release
    ```
3. Move the new chart to the docs folder:
```shell
$ mv keda-*.tgz docs

> ```

Reviewer: Can you bring this step back please?

Author: This is not required once we pass `--destination docs` option to `helm package` command. Also once I get the package renamed to `azure-cosmos-db-add-on`, the move command will need to be changed to `mv *.tgz docs` to include the Cosmos DB scaler package.

Reviewer: Nice, thank you.

> 1. Create a GitHub release for your new Helm chart version by using the following template.

Reviewer: Nit: Cna you change the list numbers back to an increasing manner (instead of 1 every time) please?

Author: IMO, this works better. We do not need to change the list numbers if a new item is inserted or deleted in between, as it happened in the PR.

> 1. Create a pull request with our new Helm index.
> 1. Create a GitHub release for your new Helm chart version by using the following template.

Reviewer: ```suggestion
5. Create a pull request with our new Helm index.
6. Create a GitHub release for your new Helm chart version by using the following template.
```
